requirements.txt

```diff
@@ -0,0 +1,61 @@
+asgiref==3.2.10
```

Comment:
this list seems very long - I suspect this is because you're also getting dependencies from the Ubunto built-in packages. Please try to use a Python [virtualenv](https://docs.python.org/3/tutorial/venv.html).

Reply:
so i tries really hard to implement it with pipenv and stack with 2 major problems:
1. i had the instructions that i can add to the readme.md and its work manually but its not work automate in bootstrap (show bellow what i did).
2. even manually there is a specific problem with library psycopg2, i succeed to install it without pipenv but when we work with virtualenv its cant find some file and fail (i think we need to define PATH in script but all the serach i try to do didnt find a solution for that )

we felt that in this point and the level of knowledge of our team its would be better to leave it like this because its work fine, even though its not the best solution.. 

bootstrp.sh- 
#!/usr/bin/env bash

# Installing packages
apt update
apt install -y python3.7 python3-pip postgresql postgresql-contrib
sudo apt install python3-dev libpq-dev

**pip3 install pipenv**
pip3 install --upgrade pip
**shopt -s cdable_vars
export PYTHONPATH=/vagrant/givitsite/
cd PYTHONPATH # i saw that in this way it can run not in the child process?
pipenv install # i fail to run this line in the script, work manually
pipenv shell # same fail, work manually
pip3 install -r /vagrant/requirements.txt**

# DB creation
sudo -u postgres createdb givit
sudo -u postgres psql -c "ALTER ROLE postgres WITH PASSWORD 'givit';"

Reply:
`pipenv` is not `virtalenv` - you seem to be mixing the two... and in the code above as well. you can use `virtualenv` without pipenv. If you use `pipenv` it wraps arounf `pip` and `virtualenv` and you should not be using them directly in that case.

If you want to use `pipenv` with Vagrant your setup should simply be:

1. install pipenv itself (with `pip install -u`)
2. Run `pipenv sync`

Then you manually setup your dependencies with `pipenv install` (In your Git clone), create the `Pipfile` and Pipfile.lock` files and commit them to Git.

Reply:
ok i guess that we need to dive again and find the solution, but right now we as a team think that we need to move forword. is that ok to you that i will open tickt for it and merge this temporary solution? even that requirement.txt include more lib cause its work fine..

Reply:
sure, open a ticket and link it here - I'm jut afraid that something in this long list will break if you'll update ubuntu or some of your dependencies....

Reply:
done. can i merge? 

thanks!